XROLE/curtainAndBlind
# #166458828: Setup testing environment

*What does this PR do?*
- This PR set up an enviroment to be able to write test

*Description of the task to be completed*
- [x] install mocha, chai, Istanbul, coveralls, mocha-lcov-reporter
- [x] add a test script
 - [x] add coverage script
 - [x] create a dummy function in utils.js
 - [x] write a test for one of the dummy function to test the implementation

*How should this be manually tested?*
- run `git fetch` to update your local branch
- run `git checkout ch-setup-testing-environment-#166458828` to navigate into the recent branch
- run `npm run test` to see the set of passing tests and their coverage report
- add more test and run `npm run test`

*Any background story to be provided*
- N/A

*What are the relevant Pivotal tracker Stories?*
[#166458828](https://www.pivotaltracker.com/story/show/166458828)
